src/styles/global.scss

```diff
@@ -104,4 +104,78 @@
     transparent 6px
   );
   background-size: 60px 60px;
+}
+
+.tiptap {
+  hr {
+    border-top-style: double;
+  }
+
+  ul[data-type="taskList"] {
+    padding-left: 17px;
+    li {
+      list-style-type: none;
+      display: flex;
+      gap: 8px;
+    }
+  }
+
+  img {
+    max-width: 100%;
+    &.ProseMirror-selectednode {
+      outline: 3px solid rgba(0, 65, 198, 0.8);
+    }
+  }
+
+  table {
+    border-collapse: collapse;
+    margin: 0;
+    overflow: hidden;
+    table-layout: fixed;
+    width: 100%;
+
+    td {
+      border-color: var(--mantine-color-gray-5) !important;
+      border-width: 1px !important;
+      border-style: solid !important;
+      box-sizing: border-box;
+      min-width: 1em;
+      padding: 3px 5px;
+      position: relative;
+      vertical-align: top;
+
+      > * {
+        margin-bottom: 0;
+      }
+    }
+
+    .selectedCell:after {
+      background: rgba(200, 200, 200, 0.4);
```

Comment:
Are you sure this is correct, doesn't need to be changed depending on the color scheme ?

Reply:
No this is just grey with opacity when you select a cell or multiple ones. Having it colored would clash with a cell that already has a background color and would look weird.
![image](https://github.com/ajnart/homarr/assets/26098587/d7804e8f-7a8d-406e-a971-5ca09d085b15)
See bottom row